### PR TITLE
chore: Bottom drawer overlay transition

### PR DESCRIPTION
**Pull request by Chef-Yogi**
before

https://user-images.githubusercontent.com/99634186/157300749-b41ef4fd-2133-4004-aca6-54c78c58bd0c.mp4

after

https://user-images.githubusercontent.com/99634186/157300767-ab746798-9215-4585-9ba8-4a7fe5c66033.mp4

### CODE REVIEW

File: packages/uikit/tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "./src",
```
Comment (chefilip) [inline packages/uikit/tsconfig.json:3]: Hmm. We are going to use the same tsconfig in `packages` folder when we are moving to `monorepo`. Not sure if this condig is still compatible with it

Comment (Chef-Yogi) [reply packages/uikit/tsconfig.json:3]: Ok, I think I can use relative import for now

**Comment by chef-jojo** [comment]
z index 

https://user-images.githubusercontent.com/94336009/157581413-06723555-85f4-47a7-af04-a06b00a4116c.mp4

**Comment by Chef-Yogi** [comment]
@chef-jojo fixed!

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**